### DISCUSSION TITLE
Getting started with notice2

### DISCUSSION THREAD

**Discussion by leofannar** [OP-TED/eForms-SDK#767] [Q&A] [answered]
Hey!
I am making a website that need to call the /api/v2/notive/notices/submit but I dont know what xml im suppose to be creating and sending.
I have found out that all the fields are here: https://github.com/OP-TED/eForms-SDK/blob/develop/fields/fields.json
So my question is, do I need to find out what single id im suppose to be using and make xml based on that from fie fields.json file or can I follow one of the examples there to make the XML, I am so confused on what and how the XML is suppose to be.

Thanks!

**Comment by YvesJo** [comment, marked as answer]
Hi,
I guess you meant "Getting started with eForms". The SDK is what you will need for that.
You may go to https://docs.ted.europa.eu/eforms/latest/guide/index.html as a starting page where you'll find links to further detailed information that will help you with the implementation.
KR,
